doc/source/_static/numpy.css

```diff
@@ -0,0 +1,137 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap');
+
+.main {
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+}
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 16px;
+  line-height: 150%;
+}
+
+div.top-scipy-org-logo-header {
+  background-color: #fafafa;
+  border-bottom: 2px solid #013243; /* warm black */
+  margin-top: 0;
+  box-shadow: none;
+}
+div.top-scipy-org-logo-header img {
+  height: 110px;
+  padding-left: 27px;
+}
+
+div.spc-navbar .nav-pills > li > a {
+  background-color: #4d77cf; /* han blue */
+}
+
+div#spc-section-body h1, h2, h3, h4, h5, h6 {
+    font-family: 'Lato', sans-serif;

```

Comment:
Nit: would be good to pick either 2 or 4 space indents here

Reply:
You're right; fixed.